hangman.py

```diff
@@ -0,0 +1,240 @@
+# Problem Set 2, hangman.py
+# Name:Mizin Danilo
+# Collaborators:Piustonen
+# Time spent:
+
+import string
+import random
+
+
+WORDLIST_FILENAME = "words.txt"
+LIST_VOWELS = frozenset("aeiou")
+STAR = "*"
```

Comment:
Is it really just a "star"? Or maybe it has some different meaning in your game?